pom.xml

```diff
@@ -118,7 +118,15 @@
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>script-security</artifactId>
         </dependency>
-
+        <dependency>
+            <groupId>io.jenkins.plugins</groupId>
+            <artifactId>markdown-formatter</artifactId>
+            <version>95.v17a_965e696ee</version>
```

Comment:
Not in `bom`? I would omit this one and stick to the more standard `antisamy-markup-formatter`.

Reply:
The antisamy-markup-formatter can only sanitize html and is not able to convert markdown to html. So without this dependency it is not possible to use markdown files for documentation (and I think we should avoid directly setting a dependency to the commonmark java library when we have a plugin that provides it). Maybe we can add it to the bom

Reply:
> without this dependency it is not possible to use markdown files for documentation

Right, I was proposing to just drop that feature.

> Maybe we can add it to the bom

You could. I have no idea how well supported this plugin is.

Reply:
Mark Waite is maintaining the plugin, so should be well supported

Reply:
I really like the markdown formatter plugin and plan to continue maintaining it.  I have it installed on my local Jenkins instances and use it regularly

Reply:
I'm happy to submit the plugin to the BOM.  It has less than 1000 installations but it is very helpful and has worked well in all the cases where I've used it.

Reply:
Submitted for possible inclusion in the plugin bill of materials:

* https://github.com/jenkinsci/bom/pull/3090

Reply:
The markdown formatter plugin is included in plugin bill of materials [2961.v1f472390972e](https://github.com/jenkinsci/bom/releases/tag/2961.v1f472390972e) (April 8, 2024).